src/Stryker.CLI/Stryker.CLI/StrykerCLI.cs

```diff
@@ -6,6 +6,19 @@
 
 namespace Stryker.CLI
 {
+    internal static class Helpers
```

Comment:
I don't think this has to be a static helper class. This could be a private method of the class `StrykerCLI`